qa/integration/src/test/java/org/eclipse/kapua/integration/misc/XmlServiceEventMarshalerTest.java

```diff
@@ -0,0 +1,105 @@
+        xmlServiceEventMarshaler = new XmlServiceEventMarshaler();
+        stringWriter = new StringWriter();
+    }
+
+    @Test
+    public void marshalXmlWithoutContextTest() throws ServiceEventBusException {
+        stringWriter.write("<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+                "<serviceEvent/>\n");
+        String expectedValues = stringWriter.toString();
+        XmlUtil.setContextProvider(new TestJAXBContextProvider());
+        assertEquals("not_equals", expectedValues, xmlServiceEventMarshaler.marshal(serviceEvent));
+    }
+
+    @Test
+    public void marshalJsonWithContextTest() throws ServiceEventBusException {
+        stringWriter.write("<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+                "<serviceEvent>\n" +
+                "   <id>id</id>\n" +
+                "   <contextId>contextId</contextId>\n" +
+                "   <entityType>entityType</entityType>\n" +
+                "   <status>SENT</status>\n" +
+                "   <note>note</note>\n" +
+                "</serviceEvent>\n");
+
+        String expectedValues = stringWriter.toString();
+        serviceEvent.setId("id");
+        serviceEvent.setContextId("contextId");
+        serviceEvent.setEntityType("entityType");
+        serviceEvent.setStatus(ServiceEvent.EventStatus.SENT);
+        serviceEvent.setNote("note");
+
+        XmlUtil.setContextProvider(new TestJAXBContextProvider());
+        assertEquals("not_equals", expectedValues, xmlServiceEventMarshaler.marshal(serviceEvent));
+    }
+
+    @Test(expected = ServiceEventBusException.class)
+    public void unmarshalXmlWithoutJAXBContextProviderTest() throws KapuaException {
+        xmlServiceEventMarshaler.unmarshal("message");
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void unmarshalXmlWithNullContextTest() throws Exception {
+        xmlServiceEventMarshaler.unmarshal(null);
+    }
+
+    @Test
+    public void unmarshalXmlWithContextTest() throws KapuaException {
+        XmlUtil.setContextProvider(new TestJAXBContextProvider());
+        ServiceEvent elements = xmlServiceEventMarshaler.unmarshal("<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
+                "<serviceEvent>\n" +
+                "   <id>id</id>\n" +
+                "   <contextId>contextId</contextId>\n" +
+                "   <entityType>entityType</entityType>\n" +
+                "   <status>SENT</status>\n" +
+                "   <note>note</note>\n" +
+                "</serviceEvent>\n");
+
+        assertEquals("not_equals", "id", elements.getId());
+        assertEquals("not_equals", "contextId", elements.getContextId());
+        assertEquals("not_equals", "entityType", elements.getEntityType());
+        assertEquals("not_equals", ServiceEvent.EventStatus.SENT, elements.getStatus());
+    }
+
+    @Test
+    public void getContentTypeTest() {
+        assertEquals("not_equals", "application/xml", xmlServiceEventMarshaler.getContentType());
```

Comment:
Please change message to "Expected and actual values are not the same." or something similar.